src/providers/twitch/TwitchChannel.cpp

```diff
@@ -621,7 +597,7 @@ const QString &TwitchChannel::popoutPlayerUrl()
 
 int TwitchChannel::chatterCount()
 {
-    return this->chatterCount_;
+    return this->accessChatters()->getNumChatters();
```

Comment:
We still can't use `getNumChatters` here since it relies on the chatters that are part of the `ChatterSet` and the `ChatterSet` a) doesn't remove stale entries b) doesn't go beyond 2000. So keeping and using the `chatterCount_` for this specific purpose still makes sense imo

Reply:
It now sets `chatterCount_` whenever `refreshChatters` is run and this function now uses `chatterCount_` again.